src/torrent.cpp

```diff
@@ -11579,6 +11579,7 @@ namespace libtorrent
 	{
 		if (!m_apply_ip_filter) return;
 		if (!m_peer_list) return;
+		if (!m_ip_filter) return;
```

Comment:
I don't understand this. If we remove the filter, we still need to tell everyone about it